src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!--  Big -->
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <label class="search">
+
+        <img
+          class="image"
+          src="images/Search.svg"
+          alt="image"
+          data-qa="big"
+        >
+
+        <input
+          class="input"
+          type="text"
+          name="text"
+          autocomplete="off"
+          placeholder= "Try &quot;Los Angeles&quot;"
+          data-qa="big keypress"
+        >
+      </label>
+
+      <label class="search search--small">
+
+        <img
+          class="image--small"
+          src="images/Search.svg"
+          alt="image"
+          data-qa="small"
+        >
+
+        <input
+          class="input--small"
```

Comment:
```suggestion
          class="input input--small"
```
Let's set some common class to avoid repeating the same styles on similar components